Resources/Prototypes/Catalog/Research/technologies.yml

```diff
@@ -104,6 +104,7 @@
   - MedkitAdvanced
   - MedkitRadiation
   - MedkitCombat
+  - FelinidCone
```

Comment:
The R&D rework obsoleted this file, so you can just move it to a static recipe on the medical techfab.

Reply:
The techfab locations don't seem to be straight forward, I might just fallback to just adding it to medidrobe

Reply:
What is not straightforward about it? All of the lathes are in the two `Structures/Machines/lathe.yml` files. There's no reason why you couldn't put it there.

https://github.com/Nyanotrasen/Nyanotrasen/blob/b026d8d17437e72d54866df67cd717fe0583cfc9/Resources/Prototypes/Entities/Structures/Machines/lathe.yml#L371-L394

Reply:
It's not in there?

Reply:
Merge master.